src/fluid/con2prim_robust.hpp

```diff
@@ -429,7 +435,8 @@ class ConToPrim {
 
     num_nans = std::isnan(v(crho)) + std::isnan(v(cmom_lo)) + std::isnan(v(ceng));
 
-    if (num_nans > 0 || res.used_gamma_max()) {
+    // if (num_nans > 0 || res.used_gamma_max()) {
```

Comment:
I think the gamma max ceiling can be safely caught subsequently by `ApplyFloors` which will limit us to `gamma_max`